packages/terra-time-input/src/TimeInput.scss

```diff
@@ -1,40 +1,204 @@
-// stylelint-disable declaration-property-value-blacklist
+// stylelint-disable declaration-property-value-blacklist scss/at-mixin-pattern
+// stylelint-disable scss/at-mixin-pattern
+
 :local {
-  [dir=ltr],
-  [dir=rtl] {
-    .time-input {
-      background-color: transparent;
-      border: #dedfe0 solid 1px;
-      border-bottom-left-radius: 0.25em;
-      border-bottom-right-radius: 0.25em;
-      border-top-left-radius: 0.25em;
-      border-top-right-radius: 0.25em;
-      direction: ltr;
-      display: inline-block;
-      font-size: 1em;
-      margin: 0;
-      padding: 0.1em 0;
+  @mixin focus-styles {
+    border-color: var(--terra-time-input-focus-keyboard-border-color, #4cb2e9);
+    border-style: var(--terra-time-input-focus-keyboard-border-style, solid);
+    border-width: var(--terra-time-input-focus-keyboard-border-width, 1px);
+    opacity: var(--terra-time-input-focus-keyboard-opacity, 1);
+    outline: none;
+  }
+
+  .time-input {
+    background-color: var(--terra-time-input-background, #fff);
+    background-image: var(--terra-time-input-focus-animation-background, none);
```

Comment:
Looks like there are some CSS custom properties in the `.time-input` selector which include the word `focus`. Not sure if this is what you meant to include. e.g I'd expect `background-image: var(--terra-time-input-focus-animation-background, none);` to actually be `background-image: var(--terra-time-input-animation-background, none);` and things like `border-style: var(--terra-time-input-desktop-focus-border-style, solid);` to actually be `border-style: var(--terra-time-input-desktop-border-style, solid);`

Then you can use `-focus-` in the CSS custom properties within the `is-focused` selector below.

Reply:
Done. https://github.com/cerner/terra-core/pull/1027/commits/1d59ce24cd77ec2a62eaeb02e086047c01bdf25a